cni/network/network_windows.go

```diff
@@ -44,10 +44,15 @@ func handleConsecutiveAdd(args *cniSkel.CmdArgs, endpointId string, nwInfo *netw
 		log.Printf("[net] Found existing endpoint through hcsshim: %+v", hnsEndpoint)
 		log.Printf("[net] Attaching ep %v to container %v", hnsEndpoint.Id, args.ContainerID)
 
-		err := hcsshim.HotAttachEndpoint(args.ContainerID, hnsEndpoint.Id)
-		if err != nil {
-			log.Printf("[cni-net] Failed to hot attach shared endpoint[%v] to container [%v], err:%v.", hnsEndpoint.Id, args.ContainerID, err)
-			return nil, err
+		endpoint, _ := hcsshim.GetHNSEndpointByID(hnsEndpoint.Id)
+		isAttached, _ := endpoint.IsAttached(args.ContainerID)
+		// Attach endpoint if it's not attached yet. If attached, populate result directly should be good enough.
```

Comment:
change to // Attach endpoint if it's not attached yet

and move above line 47, since that's what the couple lines after does